clippy.toml

```diff
@@ -0,0 +1,10 @@
+cyclomatic-complexity-threshold = 30
+doc-valid-idents = [
+  "MiB", "GiB", "TiB", "PiB", "EiB",
+  "DirectX", "OpenGL", "TrueType",
+  "GPLv2", "GPLv3",
+  "GitHub",
+  "IPv4", "IPv6",
+  "JavaScript", "NaN", "OAuth",
+  "SQLite", "PostgreSQL"
```

Comment:
What's this array actually doing? Are we sure we want to allow this? For example I named everything `Mysql` instead of `MySQL` to be ideomatic. I would assume we want the same to apply to other backends.